openfasoc/generators/common/simulation/classify_sim_error.py

```diff
@@ -0,0 +1,55 @@
+import warnings
+import sys
+import os
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from common.get_ngspice_version import check_ngspice_version
+            
+def classify_sim_error(template_file, result_file, _generator_is) -> str:
+    """Used to propogate out how close the generated simulations results are 
+    from the results in the stored template files
+
+    Args:
+    - 'template_file' (string): The stored template file
+    - 'result_file' (string): The result file generated by the simulations
+    - '_generator_is' (dict): Contains info about the generator for which the simulation check is being run
```

Comment:
Can the format of this be documented in more detail? Also, an `_` is usually only appended to variables or functions that are only used internally. Here, it would make more sense to name it `generator_is` or `generator_type` or simply `generator`.

Reply:
Fixed